nemo/collections/common/parts/rnn.py

```diff
@@ -188,10 +204,15 @@ def __init__(
                     bias.data[hidden_size : 2 * hidden_size].fill_(forget_gate_bias)
                 if "bias_hh" in name:
                     bias = getattr(self.lstm, name)
-                    bias.data[hidden_size : 2 * hidden_size].fill_(0)
+                    # bias.data[hidden_size : 2 * hidden_size].fill_(0)
+                    bias.data[hidden_size : 2 * hidden_size] *= float(hidden_hidden_bias_scale)
```

Comment:
How about to check and if hidden_hidden_bias_scale is 0.0 then skip this part?

Reply:
Is it correct that you are multiplying by zero here as the default.?
 Just to make sure it is on purpose.

Reply:
It's done just once at init, so its fast enough. The default here is matching what was done in general prior to this commit. Mathematically it is equivalent.